packages/components/collapsible-panel/src/collapsible-panel.styles.tsx

```diff
@@ -83,22 +98,34 @@ const HeaderControlsWrapper = styled.div`
   cursor: auto;
 `;
 
-const SectionContent = styled.div`
+const SectionContent = styled.div<Pick<TCollapsiblePanel, 'condensed'>>`
   width: 100%;
   display: flex;
   flex-wrap: wrap;
   align-items: flex-start;
 `;
 
-const SectionWrapper = styled.div`
-  border-top: 1px solid ${designTokens.colorNeutral60};
+const SectionDescriptionWrapper = styled('div')<{ condensed?: boolean }>`
```

Comment:
nit:
```suggestion
const SectionDescriptionWrapper = styled.div<{ condensed?: boolean }>`
```

Reply:
Updated in this commit: b2c37088f4d207992c3a579236e88828daa233b6